runtime/calamari/src/lib.rs

```diff
@@ -724,6 +742,7 @@ construct_runtime!(
 		} = 1,
 		Timestamp: pallet_timestamp::{Pallet, Call, Storage, Inherent} = 2,
 		ParachainInfo: parachain_info::{Pallet, Storage, Config} = 3,
+		TransactionPause: pallet_tx_pause::{Pallet, Call, Storage, Event<T>} = 4,
```

Comment:
Not sure which module index we should input here. @stechu, any idea?

Reply:
I would suggest to use larger rather than smaller number.